ui/src/app/(dashboard)/newstream/FunctionList.tsx

```diff
@@ -0,0 +1,25 @@
+import statusStyles from '@/utils/statusStyles';
+
+export default function FunctionList({ row }) {
+  const { functions } = row?.original;
```

Comment:
**suggestion:** Since this component only uses `functions`, it doesn't need to receive the whole `row`.

```suggestion
type FunctionListProps = {
  functions: {
    id: string;
    name: string;
    status: FunctionRunStatus;
  }[]
}

export default function FunctionList({ functions }: FunctionListProps) {
```